packages/lit-dev-content/site/tutorials/content/svg-templates/00.md

```diff
@@ -0,0 +1,14 @@
+SVG enables artists to create scalable graphics through an XML-like syntax. It's supported in most modern browsers.
+
+But beautiful graphics can get complicated!
+
+Webcomponents are a perfect match for SVG. They can encapsulate styles,
+logic, and document structure.
+
+Lit can maintain SVG through tempaltes and template fragments.
+They provide a declarative way to programmatically generate complex
+graphics.
+
+After completing this tutorial, the reader will be familiar with
+the basics of rendering SVG with Lit by creating the graphic found in this
+demo.
```

Comment:
I recommend a prior knowledge section that links out to docs and MDN for things they should know but you won't cover. e.g.

```md
## Prior Knowledge

- [styling in lit](lit docs link)
- [basic lit templating](lit docs link)
- [SVG elements](mdn link)
- [custom properties](mdn or docs link)
```

Reply:
sounds good! done